### DISCUSSION TITLE
Feature request: Servo support?

### DISCUSSION THREAD

**Discussion by quack3d** [TillFleisch/ESPHome-Philips-Smart-Coffee#59] [Q&A]
Have you considered adding support for a servo like the one from DivanX10's project? https://github.com/DivanX10/ESP-Philips-5400-Coffee-Machine/blob/main/Config/ESPHome/coffee-philips-5400-servo-en.yaml

And btw: The PCB you made was really cool but wouldn't it be possible to make one that doesn't require acquiring the picoflex cables? Something that plugs into the display and stays there?

**Comment by TillFleisch** [comment]
You should be able to use [ESPHome's servo component](https://esphome.io/components/servo.html) in combination with local [automations](https://esphome.io/guides/automations.html)/scripts to achieve this. You may use the status sensor as a trigger for your automations. The linked configuration file also uses the `servo` component.

> make one that doesn't require acquiring the picoflex cables? 

Maybe? I am not aware of any female SMD/Throughole connectors to which a male picoflex connector can be connected, such that the custom pcb can sit at the back side of the display. Since I wanted to keep things simple for my first PCB, I wen't with the "easy route" of using two male picoflex connectors and a short cable.
